.babelrc

```diff
@@ -1,4 +1,13 @@
 {
+  "sourceMaps": true,
```

Comment:
Source maps are output for debugging. They are only generated when the `--source-maps` option is passed to the babel cli.

Currently only the gatsby lib creates source maps like this. The packages don't because most of them build to `.` and this would create noise. (or have to add .map to gitignore). This will be refactored soon anyway so I excluded the other packages for now.

Optionally we could use the "env" config if different settings are required for production.